docs/source/verdi/verdi_user_guide.rst

```diff
@@ -26,155 +48,112 @@ For example ``verdi process kill --help`` will display:
+.. note::
 
-Consider for example a database with the following three groups:
-
-===  =====================================  ========
-ID   UUID                                   LABEL
-===  =====================================  ========
-10   12dfb104-7b2b-4bca-adc0-1e4fd4ffcc88   group
-11   deadbeef-62ba-444f-976d-31d925dac557   10
-12   3df34a1e-5215-4e1a-b626-7f75b9586ef5   deadbeef
-===  =====================================  ========
+   PKs are easy to type and work as long as you stay within your database.
+   **When sharing data with others, however, always use UUIDs.**
 
-We would run into trouble if we wanted to identify the second group by its label ``10``, since it would first be interpreted as an ``ID``, which would return the first group instead.
-Likewise, if we wanted to retrieve the third group by its label, we would get the second group instead, since the label ``deadbeef`` is also a valid partial UUID of the second group.
-Finally, say we wanted to select the first group using its partial ``UUID`` ``12``, it would unfortunately match the third group on its ``ID`` instead.
+Any ``verdi`` command that expects an identifier as a paramter will accept PKs, UUIDs and labels.
 
-Luckily, ``verdi`` provides the tools to break all of these ambiguities with guaranteed success.
-The latter ambiguity, between an ``ID`` and ``UUID`` can always be resolved by passing a larger partial ``UUID``.
-Inevitably, eventually a non-numeric character or a dash will be included in the partial ``UUID``, rendering it an invalid ``ID`` and the identifier will be cast to the right type.
-The case of an identifier, that is intended to refer to a ``LABEL``, that just happens to also be a valid ``ID`` or ``UUID`` cannot be solved in this way.
-For this case ``verdi`` reserves a special character, the exclamation mark ``!`` that can be appended to the identifier.
-Before any type guessing is done, the command line will check for the presence of this marker, and if found will directly interpret the identifier as a ``LABEL``.
-For example, to solve ambiguity problems of the first two examples given in this section, one would have had to pass ``10!`` and ``deadbeef!``.
-The exclamation point would have forced them to be interpreted as a ``LABEL`` and ensured that the right group would be retrieved.
+In almost all cases, this will work out of the box.
+Since command line parameters are passed as strings, AiiDA needs to deduce the type of identifier from its content, which can fail in edge cases (see :ref:`cli_identifier_resolution` for details).
+You can take the following precautions in order to avoid such edge cases:
 
-In summary, to guarantee correct identification of a specific type:
+  * PK: no precautions needed
+  * UUID: no precautions needed for full UUIDs. Partial UUIDs should include at least one non-numeric character or dash.
```

Comment:
```suggestion
  * UUID: no precautions needed for full UUIDs. Partial UUIDs should include at least one non-numeric character or dash
```

You don't have an ending period in any of your other lists (as far as I remember).